sdk/src/types/index.ts

```diff
@@ -27,6 +27,7 @@ export type Attestation = OnChainAttestation & {
   schemaString: string;
   decodedData: string[];
   decodedPayload: object;
+  offchainData?: { schemaId: string; uri: string; error?: string };
```

Comment:
Can you extract the `{ schemaId: string; uri: string; error?: string }` part in a dedicated type?

Reply:
Done